src/autopas/selectors/tuningStrategy/ActiveHarmony.h

```diff
@@ -29,30 +38,33 @@ class ActiveHarmony : public TuningStrategyInterface {
    * @param allowedLoadEstimatorOptions
    * @param allowedDataLayoutOptions
    * @param allowedNewton3Options
+   * @param mpiStrategyOption
+   * @param comm Default value shouldn't be used. Only provided to not have to set it as first parameter.
    */
   ActiveHarmony(const std::set<ContainerOption> &allowedContainerOptions = ContainerOption::getAllOptions(),
```

Comment:
The tests also call the constructor. Since I generally use ```mpi.h``` in the tests and ```MPIWrapper.h``` inside of AutoPas, I need the default values to be able to call the constructor at all.

Reply:
Just tested this:
removing ALL defaults for this constructor and adding MPI_COMM_WORLD (or AUTOPAS_MPI_COMM_WORLD) as last argument for the constructor (make_unique) in `MPIParallelizedStrategyTest.cpp` works.